pkg/logging/config.go

```diff
@@ -50,13 +50,11 @@ func NewLogger(configJSON string, levelOverride string) (*zap.SugaredLogger, zap
 		}
 	}
 
-	logger, err2 := loggingCfg.Build()
+	logger, err2 := loggingCfg.Build(zap.Fields(zap.String("fallback-logger", "true")))
```

Comment:
Thanks for the changes. Overall, think looks good. One nit: Can you used a named logger rather than adding an extra field to be consistent with what we do here at: https://github.com/knative/pkg/blob/03fb14296d9a79df7743e7037e381846a57f7ff1/logging/logger.go#L39